students/rachel_s/session03/mailroom.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+import csv
+
+# Import donor data
+
+donor_info = []
+donor_names = []
+
+def add_donation(db, name, amount):
+    for item in db:
+        if item[0] == name.lower():
+            item[1].append(amount)
+
+def add_new_donor(db, name, amount):
+    db.append([name, [amount]])
+
+def get_donor(db, name):
+    for item in db:
+        if item[0] == name.lower():
+            return item
+
+with open('donors.csv') as csv_file:
+    csv_reader = csv.reader(csv_file, delimiter=',')
+    for line in csv_reader:
+        if not line[0].lower() in donor_names:
+            donor_names.append(line[0].lower())
+            add_new_donor(donor_info, line[0].lower(), int(line[1]))
+        else:
+            add_donation(donor_info, line[0], int(line[1]))
```

Comment:
Wow, you already implemented the file reading feature, which is covered in the next week, not this week. Excellent!